src/App.js

```diff
@@ -79,6 +83,21 @@ class App extends Component {
       dataLoadingError: error.message,
     });
   }
+  selectRoom = (roomID) => {
+    this.setState({
+      selectedRoomID: roomID,
+    });
+  }
```

Comment:
@pani-tka exists a good practice to make an empty line between methods - it improves code readability